ocaml/database/schema.ml

```diff
@@ -35,7 +35,7 @@ module Value = struct
     | String of string
     | Set of string list
     | Pairs of (string * string) list
-  [@@deriving sexp]
+  [@@deriving sexp, rpc]
```

Comment:
Are we sure we want to use `rpc` and not `rpcty` here?

Reply:
I'm not sure I understand the difference between the two sufficiently. But rpcty causes errors in the type of Value for me

Reply:
Ok, I've had a chat with Jon, and he's pointed out what the problem is, this is my (paraphrased) understanding of it.
In trying to use rpcty with Schema.Value, the auto generated code has multiple instances of `String`. `let open! Rpc.Types in` is called above each of the type options, and as such inside the definition of `String of string`, the compiler ~doesn't know~ gets the wrong `String`. This means that it's trying to apply the rpc String, where it should be applying the Schema.Value String.
I think it's something that Jon has added to the infinite 'to do' list, but in the mean time the solution is, 'Use rpc, rather than rpcty'